shell/platform/darwin/common/framework/Source/FlutterStandardCodec.mm

```diff
@@ -6,6 +6,15 @@
 
 #pragma mark - Codec for basic message channel
 
+static const UInt8 s_zeroBuffer[8] = {0, 0, 0, 0, 0, 0, 0, 0};
```

Comment:
Can you move these statics into the functions where they are called to make it more obvious?

You may want to put the initialization in a `dispatch_once` block in the functions so they aren't set until they are needed.

Reply:
I've gotten comments asking for keeping them at the top in other code reviews at google.  I double checked the style guide and I couldn't find any recommendation either way.  These won't actually get set until they are paged into memory.